app/assets/stylesheets/forever_style_guide/modules/_all.scss

```diff
@@ -17,6 +17,7 @@
 @import "nav-dropdowns";
 @import "nav-dropdowns-account";
 @import "nav-dropdowns-ambassador";
+@import "nav-dropdowns-deals";
```

Comment:
nav-dropdowns-deals is a little misleading since this isn't a dropdown. We could probably handle it more generically like nav-item-emphasized and then that style could conceivably be applied to any item we want to have the alternate styling.